lib/src/models/parsed_certificate.dart

```diff
@@ -74,19 +103,35 @@ sealed class ParsedCertificate with _$ParsedCertificate {
     required BigInt retirementEpoch,
   }) = StakePoolRetirement;
 
-  late final CertificateType certificateType = switch (this) {
-    StakeRegistration() => CertificateType.stakeRegistration,
-    StakeRegistrationConway() => CertificateType.stakeRegistrationConway,
-    StakeDeregistration() => CertificateType.stakeDeregistration,
-    StakeDeregistrationConway() => CertificateType.stakeDeregistrationConway,
-    StakeDelegation() => CertificateType.stakeDelegation,
-    VoteDelegation() => CertificateType.voteDelegation,
-    AuthorizeCommitteeHot() => CertificateType.authorizeCommitteeHot,
-    ResignCommitteeCold() => CertificateType.resignCommitteeCold,
-    DRepRegistration() => CertificateType.dRepRegistration,
-    DRepDeregistration() => CertificateType.dRepDeregistration,
-    DRepUpdate() => CertificateType.dRepUpdate,
-    StakePoolRegistration() => CertificateType.stakePoolRegistration,
-    StakePoolRetirement() => CertificateType.stakePoolRetirement,
+  late final bool isConway = switch (this) {
+    StakeRegistration() => false,
+    StakeDeregistration() => false,
+    StakeDelegation() => false,
+    StakePoolRegistration() => false,
+    StakePoolRetirement() => false,
+    StakeRegistrationConway() => true,
+    StakeDeregistrationConway() => true,
+    VoteDelegation() => true,
+    AuthorizeCommitteeHot() => true,
+    ResignCommitteeCold() => true,
+    DRepRegistration() => true,
+    DRepDeregistration() => true,
+    DRepUpdate() => true,
```

Comment:
Please double check if those are correct. I added them myself during the call based on my memory so there's a chance there may be a mistake here.

Reply:
checked the ts file and they are correct